### PR TITLE
Feature/chk return new layout

### PR DESCRIPTION
**Pull request by jaswinder2303**
Product Backlog Item 69538:Check your return: new design incl PCS breakdown

### CODE REVIEW

File: src/EA.Weee.Core.Tests.Unit/AatfReturn/AatfSchemeDataTests.cs [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    using EA.Weee.Core.AatfReturn;
+    using FakeItEasy;
+    using FluentAssertions;
+    using System;
+    using Xunit;
+
+    public class AatfSchemeDataTests
+    {
+        [Fact]
+        public void Construct_GivenNullSchemeAndObligatedReceivedData_ArgumentNullExceptionExpected()
```
Comment (SamGoodeySFW) [inline src/EA.Weee.Core.Tests.Unit/AatfReturn/AatfSchemeDataTests.cs]: Split this test into 2 tests, 1 to test the scheme and the other to test the obligated received data. Just to make sure that both are working as expected

File: src/EA.Weee.Web/Areas/AatfReturn/Mappings/ToViewModel/ReturnToReturnViewModelMap.cs [hunk 2]
```diff
@@ -47,7 +48,27 @@ public ReturnViewModel Map(ReturnData source)
                 {
                     var weeeReceivedData = source.ObligatedWeeeReceivedData.Where(s => s.Aatf.Id == aatf.Id).ToList();
                     var weeeReusedData = source.ObligatedWeeeReusedData.Where(s => s.Aatf.Id == aatf.Id).ToList();
-                    var obligatedData = new AatfObligatedData(aatf)
+
+                    var schemeData = new List<AatfSchemeData>(); // NEED TO GENERATE THIS LIST BASED OF THE weee received data and the schemeList and update the mapper tests to test this
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.Web/Areas/AatfReturn/Mappings/ToViewModel/ReturnToReturnViewModelMap.cs]: Remove the comment I made

File: src/EA.Weee.Core/AatfReturn/AatfObligatedData.cs [hunk 3]
```diff
@@ -1,27 +1,25 @@
...
-            Aatf = aatf;
-            WeeeReceived = weeeReceived;
-            WeeeReused = weeeReused;
-            WeeeSentOn = weeeSentOn;
-        }
-
-        public AatfObligatedData(AatfData aatf)
+        public AatfObligatedData(AatfData aatf, List<AatfSchemeData> schemeData)
         {
+            // GUARD AGAINST AATF and SCHEMEDATA BEING NULL AND CREATE UNIT TEST TO TEST THE GUARD
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.Core/AatfReturn/AatfObligatedData.cs]: Remove the comment I made

File: src/EA.Weee.Domain/AatfReturn/ReturnQuarterWindow.cs [hunk 4]
```diff
@@ -24,10 +24,18 @@ public class ReturnQuarterWindow
...
+        public ReturnQuarterWindow(Return @return, QuarterWindow quarterWindow, 
+            List<Aatf> aatfs, List<NonObligatedWeee> nonObligatedWeeeList, 
+            List<WeeeReceivedAmount> obligatedReceivedList, 
+            List<WeeeReusedAmount> obligatedReusedList, 
+            Operator returnOperator,
+            List<ReturnScheme> returnSchemes)
         {
             Guard.ArgumentNotNull(() => @return, @return);
             Guard.ArgumentNotNull(() => quarterWindow, quarterWindow);
+            Guard.ArgumentNotNull(() => returnSchemes, returnSchemes); // CREATE UNIT TEST TO TEST THIS Argument is not null
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.Domain/AatfReturn/ReturnQuarterWindow.cs]: Remove the comment I made

**Pull request merged**